README.md

```diff
@@ -13,13 +13,68 @@ Use [Composer](https://getcomposer.org/) to install the testing framework.
 
 ```bash
 $ composer require --dev nimut/testing-framework
+$ composer require typo3/cms-core
+$ composer require typo3/cms
```

Comment:
This is wrong and does not belong to the testing framework

Reply:
I thought it is a requirement because this testing framework is not working without typo3 core.
Also it wasn't documented.

Reply:
The reasons for not mentioning those dependencies are they don't belong directly to the usage of the testing-framework.

typo3/cms-core should be required by the extension even without the testing-framework

typo3/cms is no dependency of the extension - at least only for develop context. but in aspect of the testing-framework this dependency is added by your CI server to be flexible.

Both commands only require the latest, stable version of TYPO3 CMS. As the testing-framework is designed to test against multiple versions at the same time, I would not recommend to add those to the documentation.

Reply:
> should be required by the extension even without the testing-framework

I agree that a dependency that comes along by the unit under test, in any case, does not need to be explicitly listed. (In other frameworks this has proven to cause more issues than advantages, IIRC.)

Reply:
> this dependency is added by your CI server to be flexible.

You do assume that tests are run exclusively by the CI server? Let's agree that this is bad practice. We shouldn't presume such a setup. Tests should always be run locally first, as an engineers duty to "not ship broken code".

The CI server is meant to be an engineers colleague that doesn't forget to run _all_ the tests. We, as humans, can forget or be negligent. We may run a subset of the tests only, or be otherwise lazy. Disciplined engineers will try to avoid the latter.

Reply:
I don't asume anything. I run tests locally on my own. But then as part of another project or with "composer require typo3/cms=^x.y" without committing the dependency. And currently I don't have any problems using the extension/framework like this. Maybe you find this blog post http://insight.helhum.io/post/148886148725/composerjson-specification-for-typo3-extensions interesting. There you find a recommendation of how your extensions composer.json should look like.

By the way: only require typo3/cms doesn't help you to run your tests as your extension within this typo3 installation is still missing. You have to create a link or copy your extension files to typo3conf/ext at the end. Either manually or by using some scripts.